src/system/RoutesModule/Resources/views/Route/reload.tpl

```diff
@@ -1,39 +0,0 @@
-{include file="Admin/header.tpl"}
```

Comment:
Instead of including `header.tpl` there are now two different base templates:
`{% extends routeArea == 'admin' ? 'ZikulaRoutesModule::adminBase.html.twig' : 'ZikulaRoutesModule::base.html.twig' %}`

The custom version of the reload template has been removed,  because the reload action uses no template anymore. It has been simplified before in https://github.com/Guite/core/commit/5c062811b640918ebcaec8fb2e4a12e9f979f604